### ISSUE TITLE
[❓] How to disable one or more rules for a specific file?

### ISSUE THREAD

**Issue by claudio-salvio** [markdownlint/markdownlint#421]
Hello,

Is there any way to disable a rule or all the rules for a specific file?
For example, using some kind of comment such as ESLint allows.

Best regards,
Claudio Salvio

**Comment by zoechi** [comment]
Similar to #188 ?

**Comment by nbehrnd** [comment]
Do you search something in line of e.g., `mdl -r ~MD033,~MD034 ./example.md` to
not use criterion MD033, nor MD034?

**Comment by HariSekhon** [comment]
+1 for adding / better documenting how to exclude a line or block from a rule, for example sometimes you need to make an exemption to a rule you otherwise use, eg 

```
MD031 Fenced code blocks should be surrounded by blank lines
```

when documenting how to do markdown!!

```
````text
```shell
echo "execute this command"
...
````
